### PR TITLE
chore: copy and release periodically in `TokenService` migration

### PR DESCRIPTION
**Pull request by tinker-michaelj**
**Description**:
 - Quick-and-dirty change to ensure target `VirtualMap`'s are being regularly flushed to disk throughout the `TokenService` migration.
 - Implementation details may need some refinement.

### CODE REVIEW

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleHederaState.java [hunk 1]
```diff
@@ -110,10 +110,10 @@ public class MerkleHederaState extends PartialNaryMerkleInternal implements Merk
      */
     private static final long DO_NOT_USE_IN_REAL_LIFE_CLASS_ID = 0x0000deadbeef0000L;
 
-    private static final long CLASS_ID = 0x2de3ead3caf06392L;
+    //    private static final long CLASS_ID = 0x2de3ead3caf06392L;
     // Uncomment the following class ID to run a mono -> modular state migration
     // NOTE: also change class ID of ServicesState
-    // private static final long CLASS_ID = 0x8e300b0dfdafbb1aL;
+    private static final long CLASS_ID = 0x8e300b0dfdafbb1aL;
```
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleHederaState.java:116]: should be commented back ?

**Review by iwsimon — APPROVED** [review]
LGTM

File: hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ServicesState.java [hunk 2]
```diff
@@ -115,7 +115,7 @@ public class ServicesState extends PartialNaryMerkleInternal
         implements MerkleInternal, SwirldState, StateChildrenProvider {
     private static final Logger log = LogManager.getLogger(ServicesState.class);
 
-    private static final long RUNTIME_CONSTRUCTABLE_ID = 0x8e300b0dfdafbb1aL;
+    private static final long RUNTIME_CONSTRUCTABLE_ID = 0x8e300b0dfdafbb1bL;
```
[the remark this answers is not in the document]
Comment (Neeharika-Sompalli) [reply hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ServicesState.java:118]: fixed

File: hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleHederaState.java (hunk 1 above)
Comment (Neeharika-Sompalli) [reply hedera-node/hedera-app/src/main/java/com/hedera/node/app/state/merkle/MerkleHederaState.java:116]: fixed

File: hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/InitialModServiceTokenSchema.java [hunk 3]
```diff
@@ -184,10 +186,13 @@ public void migrate(@NonNull final MigrationContext ctx) {
 
         if (acctsFs != null) {
             log.info("BBM: migrating token service");
+            System.out.println("BBM: migrating token service");
 
             // ---------- NFTs
             log.info("BBM: doing nfts...");
-            var nftsToState = ctx.newStates().<NftID, Nft>get(NFTS_KEY);
+            System.out.println("BBM: doing nfts...");
```
[the remark this answers is not in the document]
Comment (Neeharika-Sompalli) [reply hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/InitialModServiceTokenSchema.java:193]: removed

**Pull request merged**